docker/config_template.yaml

```diff
@@ -196,14 +196,26 @@ services:
 clusterMetadata:
     enableGlobalDomain: {{ default .Env.ENABLE_GLOBAL_DOMAIN "false" }}
     failoverVersionIncrement: 10
-    masterClusterName: "master"
-    currentClusterName: "master"
+    masterClusterName: "active"
+    {{- if .Env.IS_NOT_ACTIVE }}
+    currentClusterName: "standby"
+    {{- else }}
+    currentClusterName: "active"
+    {{- end }}
     clusterInformation:
-        master:
+        active:
```

Comment:
Can we not use active/standby? I purposely changed this because this is confusing in the XDC concepts. Active/Standby is a concept for domain, not cluster. If master is not the right word, how about `clusterDCA`, `clusterDCB`? @yux0

Reply:
how about `primary` and `secondary`

Reply:
yeah that works too. I like the idea :D